doc/nrf/ug_nrf5340.rst

```diff
@@ -153,15 +154,35 @@ For example:
 
   This configuration guarantees that the network sample can handle the Bluetooth LE DLE update procedure, which is used in the :ref:`ble_throughput` sample.
 
-IEEE 802.15.4 (Thread and Zigbee)
----------------------------------
+Single protocol IEEE 802.15.4 (Thread and Zigbee)
```

Comment:
I'm not sure if `Single protocol` part is necessary here. I think if we don't use `multiprotocol` word in this context it is clear that Thread/Zigbee is single protocol